README.md

```diff
@@ -1,10 +1,23 @@
-Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type)
+Credit Card Type [![Build Status](https://travis-ci.org/braintree/credit-card-type.svg)](https://travis-ci.org/braintree/credit-card-type) [![npm version](https://badge.fury.io/js/credit-card-type.svg)](http://badge.fury.io/js/credit-card-type) [![Bower](https://badge.fury.io/bo/credit-card-type.svg)](http://badge.fury.io/bo/credit-card-type)
 ================
 
 Credit Card Type provides a useful utility method for determining a credit card type from both fully qualified and partial numbers. This is not a validation library but rather a smaller component to help you build your own validation or UI library.
 
 This library is designed for type-as-you-go detection (supports partial numbers) and is written in CommonJS so you can use it in Node, io.js, and the [browser](http://browserify.org).
 
+## Download
+To install via s/NPM/npm: 
```

Comment:
Oops sorry for the confusion, I meant swap 'NPM' out with 'npm' (I think it's [sed syntax](http://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html)?)

If you could change it to `To install via npm:` (and also potentially squash all the commits into one commit) then I think we're good to merge! :smile: